File: kotlin-maven-playground/src/test/java/org/example/kotlin/core/java/MixedLangTest.java
package org.example.kotlin.core.java;

import com.example.kotlin.maven.playground.java.model.FooBean;
import com.example.kotlin.maven.playground.model.BarBean;
import org.junit.jupiter.api.Test;

class MixedLangTest {

  @Test
  void test() throws ClassNotFoundException {
    FooBean foo = new FooBean();
    foo.setName("java222-foo");

    System.out.println(foo);

    System.out.println(Class.forName("com.example.kotlin.maven.playground.model.BarBean"));

    BarBean barBean = new BarBean();
    System.out.println(barBean.toString());
    System.out.println(barBean.getText1());
  }

}

File: kotlin-maven-playground/src/test/kotlin/com/example/kotlin/maven/playground/MixedLangTest.kt
package com.example.kotlin.maven.playground

import com.example.kotlin.maven.playground.model.BarBean
import org.junit.jupiter.api.Test

class MixedLangTest {

  @Test
  fun test() {
    println(BarBean())
    println(com.example.kotlin.maven.playground.java.model.FooBean())

    listOf(1, 2, 3).forEach { v -> println("number: $v") }
  }
}